examples/ota-provider-app/ota-provider-common/BdxOtaSender.cpp

```diff
@@ -146,8 +162,11 @@ void BdxOtaSender::HandleTransferSessionOutput(TransferSession::OutputEvent & ev
         mNumBytesSent = static_cast<uint32_t>(mNumBytesSent + blockData.Length);
         otaFile.close();
 
-        VerifyOrReturn(CHIP_NO_ERROR == mTransfer.PrepareBlock(blockData),
-                       ChipLogError(BDX, "%s: PrepareBlock failed: %s", __FUNCTION__, chip::ErrorStr(err)));
+        if (CHIP_NO_ERROR != mTransfer.PrepareBlock(blockData))
+        {
+            ChipLogError(BDX, "PrepareBlock failed: %s", chip::ErrorStr(err));
```

Comment:
This should be `CHIP_FORMAT_ERROR` instead of %s and `err.Format()`/

Also, the `err` is not updated. This should be:

```
err = mTransfer.PrepareBlock(blockData);
if (CHIP_NO_ERROR != err)
{
    ...
}
```

Reply:
Changed this as well as remaining occurrences of logging error with "%s". Also, updating the value of `err` before logging.